rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="move-to-microprofile-rest-client-1.3"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Microprofile REST Client 1.0 applications, which was upgraded to version 1.3 on JBoss EAP 7.3, and requires applications changes.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[7,8)" />        
+    </metadata>
+    <rules>
+        <!-- https://issues.jboss.org/browse/WINDUPRULE-400 -->
+        <rule id="move-to-microprofile-rest-client-1.3-00001">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver" >
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Move to Microprofile REST Client 1.3" effort="1" category-id="potential">
+                    <message>The application imports class 'org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver', and should be changed to use instead Microprofile REST Client 1.3.</message>
```

Comment:
I think it would help to mention to use `RestClientBuilder` instead.

Reply:
changed, is it ok now?

Reply:
perfect :+1: